app_flair.py

```diff
@@ -289,44 +417,63 @@ def predict_pos(prediction_request: PredictionRequest) -> PredictionResponse:
     for sen in text:
         for token in sen:
             for t in token.tags.values():
-                fields = {'begin': tokens_cas[token.idx].begin,
-                          'end': tokens_cas[token.idx].end,
-                          IS_PREDICTION: True,
-                          prediction_request.feature + "_score": t.score,
-                          prediction_request.feature: t.value}
-                annotation = AnnotationType(**fields)
+                fields = {
+                    "begin": tokens_cas[token.idx].begin,
+                    "end": tokens_cas[token.idx].end,
+                    IS_PREDICTION: True,
+                    prediction_request.feature + "_score": t.score,
+                    prediction_request.feature: t.value,
+                }
+                annotation = annotationtype(**fields)
                 cas.add_annotation(annotation)
+    return cas
+
+
+class TestBIOEncoding(unittest.TestCase):
```

Comment:
Please move this to a new folder "tests" and put it into an extra file. Add in the readme how to run the tests.

Reply:
I ran it with `python -m unittest discover` from the command line

Reply:
I ran the test dynamically. I uploaded a document with the correct labels in INCEpTION and sent a request. After BIO encoding the document, I use the unittest to compare it with the original document. In this case, I think the test can not be run directly from the command line.

Reply:
The test needs to be runnable standalone though in an isolated setting.